rfi_file_monitor/engines/directory_watchdog_engine.py

```diff
@@ -122,7 +123,7 @@ def _search_for_existing_files(self, directory: Path) -> int:
         for child in directory.iterdir():
             if child.is_file() \
                 and not child.is_symlink() \
-                and match_path(str(child), included_patterns=self._included_patterns, excluded_patterns=self._excluded_patterns,
+                and match_path(PurePosixPath(child), included_patterns=self._included_patterns, excluded_patterns=self._excluded_patterns,
```

Comment:
`child` is already an instance of `Path` here, casting won't work here I think. Also, `PurePosixPath` is only appropriate on macOS/Linux.

Reply:
strange it worked on my Windows machine... I will change it just child...